lstchain/scripts/tests/test_lstchain_scripts.py

```diff
@@ -430,6 +431,29 @@ def test_dl1ab_no_images(simulated_dl1_file, tmp_path):
             assert (new_parameters['length'] != old_parameters['length']).any()
 
 
+def test_dl1ab_on_modified_images(simulated_dl1ab, tmp_path):
+    config_path = tmp_path / 'config_image_modifier.json'
+    output_file = tmp_path / 'dl1ab_on_modified_images.h5'
+    reprocess_output_file = tmp_path / 'dl1ab_on_modified_images_reprocess.h5'
+    config_path = Path(__file__).parent.joinpath("../../data/lstchain_dl1ab_tune_MC_to_Crab_config.json")
```

Comment:
Maybe better 
```python
from importlib.resources import files
config_path = files("lstchain.data") / "lstchain_dl1ab_tune_MC_to_Crab_config.json"
```

Reply:
thank you this is indeed better, I made the change